Poker.BE/Poker.BE.API/Controllers/RoomController.cs

```diff
@@ -11,36 +11,55 @@
 
 namespace Poker.BE.API.Controllers
 {
-    public class RoomController : ApiController
-    {
-        #region Fields
-        private IRoomsService service;
-        #endregion
+	public class RoomController : ApiController
+	{
+		#region Fields
+		private IRoomsService service;
+		#endregion
 
-        #region Constructors
-        public RoomController()
-        {
-            service = new Service.Services.RoomsService();
-        }
-        #endregion
+		#region Properties
+		public IRoomsService Service { get { return service; } }
+		#endregion
 
-        #region Methods
-        [HttpPost]
-        public HttpResponseMessage EnterRoom(EnterRoomRequest request)
-        {
-            var result = default(EnterRoomResult);
+		#region Constructors
+		public RoomController()
+		{
+			service = new Service.Services.RoomsService();
+		}
+		#endregion
 
-            try
-            {
-                result = service.EnterRoom(request);
-            }
-            catch (Exception e)
-            {
-                result.ErrorMessage = e.Message;
-            }
+		#region Methods
+		[HttpPost]
+		public HttpResponseMessage EnterRoom(EnterRoomRequest request)
+		{
+			var result = default(EnterRoomResult);
 
-            return Request.CreateResponse(HttpStatusCode.OK, result);
-        }
-        #endregion
-    }
+			try
+			{
+				result = service.EnterRoom(request);
+			}
+			catch (Exception e)
+			{
+				result.ErrorMessage = e.Message;
```

Comment:
The type 'ILogger' is defined in an assembly that is not referenced. You must add a reference to assembly 'Poker.BE.Domain ... לפי מה שהבנתי לא אמורים לגעת בדומיין דרך הקונטרולרים